.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,44 @@
+# Bump Version workflow that is triggered manually
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      pcluster-version:
+        description: 'The target version of ParallelCluster CLI'
+        required: true
+        type: string
+      branch:
+        description: 'The Github branch name'
+        required: true
+        type: string
+
+jobs:
+  create-pull-requests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: ${{ inputs.branch }}
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - run: |
+        sudo npm install -g redoc-cli
+        sudo snap install yq
+    - name: Modifiy Code to Change version
+      run: ./util/bump-version.sh  --version ${{ inputs.pcluster-version }}
+        
+    - name: Create a Pull Request
+      uses: peter-evans/create-pull-request@v6
```

Comment:
Are we sure we want to use a 3rd party git-repo and not this one https://github.com/marketplace/actions/github-action-for-creating-pull-requests?

Reply:
Found the link for the one you were using https://github.com/marketplace/actions/create-pull-request